lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -649,6 +649,8 @@ def __call__(self, image: List[Image.Image], threshold: float) -> List[List[Elem
 
 
 class DeformableDetr(SycamoreObjectDetection):
+
+    @requires_modules("transformers", extra="local-inference")
```

Comment:
transformers gets pulled in by sentence-transformers then?

Reply:
Yes, but why is that relevant here? We only use sentence-transformers in the embedder, right? (Not disagreeing, just trying to understand if I missed something).

Reply:
bringing it up cuz I didn't see it marked as a dep in the extra

Reply:
Oh yeah. It seems like we never had an explicit dependency on transformers, so I never moved it. Probably worth adding. Will do.